### ISSUE TITLE
Error while parsing appveyor file

### ISSUE THREAD

**Issue by jerriep** [joaope/LocalAppVeyor#12]
When running the `jobs` command, I get the error "Error while parsing 'C:\Development\jerriep\github-issues-cli\appveyor.yml' file. Build aborted."

![image](https://user-images.githubusercontent.com/1006420/40762267-63226c72-64ca-11e8-9453-f553d0557ab7.png)

The appveyor file in question can be found here:
https://github.com/jerriep/github-issues-cli/blob/master/appveyor.yml

**Comment by joaope** [comment]
Thank you for reporting. It should be fixed with [latest release](https://github.com/joaope/LocalAppVeyor/releases/tag/v0.5.0-alpha8).

Problem was with the `global` block inside `environment`. I wasn't aware of that keyword but it's definitely valid. It's supported for _[historical reasons](https://help.appveyor.com/discussions/questions/2172-meaning-of-global-in-environment-of-appveyoryml#comment_43979045)_ as top-level variables are always considered global without the need to declare it explicitly.

So basically this:
```yml
environment:
  global:
    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
    DOTNET_CLI_TELEMETRY_OPTOUT: 1
```

is the same as this:
```yml
environment:
  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
  DOTNET_CLI_TELEMETRY_OPTOUT: 1
```

Both scenarios supported with latest.
